### PR TITLE
Add telemetry for emulation states

### PR DESCRIPTION
**Pull request by hkal**

### CODE REVIEW

File: src/screencastPanel.ts [hunk 1]
```diff
@@ -137,6 +139,19 @@ export class ScreencastPanel {
...
     }
 
+    private onSocketTelemetry(message: string) {
+        const telemetry: TelemetryData = JSON.parse(message) as TelemetryData;
+        if (telemetry.event !== 'screencast') {
+            return;
+        }
+
+        const properties: ITelemetryProps = {};
+        properties[telemetry.name] = JSON.stringify(telemetry.data);
```
Comment (antross) [inline src/screencastPanel.ts:149]: @hkal I see this is based on how we submit errors, but will we be able to query / create summaries based on these stringified pieces of data?

```
devtools/Screencast.Emulation: {"Screencast.Emulation":"{\"event\":\"forcedColors\",\"value\":\"active\"}"}, undefined
```

Comment (mliao95) [reply src/screencastPanel.ts:149]: Similar comment.  Would prefer to have the data formatted like this:
```
devtools/Screencast.Emulation: {"event":"forcedColors","value":"active"}, undefined
```

**Comment by mliao95** [comment]
Taking another look at this, I realize that the telemetry event structure is kind of odd.

We shouldn't be categorizing a telemetry event using an "event" property, it should be part of the event itself.

For instance, I would imagine an event like this:

```
devtools/Screencast.Emulation: {"event":"emulatedMedia","value":"screen"}, undefined
```

should be structured like this:
```
devtools/Screencast.Emulation/emulatedMedia: {"value":"screen"}, undefined
```

**Review by mliao95 — APPROVED** [review]
LGTM.  Thanks for the changes.

**Pull request merged**